### PR TITLE
Updated cosmos docs and code snippets

### PR DESCRIPTION
**Pull request by kushagraThapar**
* Updated cosmos docs
* Updated change feed processor code snippets

### CODE REVIEW

File: sdk/cosmos/azure-cosmos/src/samples/java/com/azure/cosmos/ChangeFeedProcessorCodeSnippet.java [hunk 1]
```diff
@@ -2,25 +2,51 @@
...
 /**
  * Code snippets for {@link ChangeFeedProcessor}
  */
 public class ChangeFeedProcessorCodeSnippet {
 
     public void changeFeedProcessorBuilderCodeSnippet() {
-        String hostName = null;
-        CosmosAsyncContainer feedContainer = null;
-        CosmosAsyncContainer leaseContainer = null;
+        String hostName = "test-host-name";
```
Comment (milismsft) [inline sdk/cosmos/azure-cosmos/src/samples/java/com/azure/cosmos/ChangeFeedProcessorCodeSnippet.java:13]: If we reference this in the docs, we should make sure that we capture that this string must be unique like suffix this with a random generated string

File: sdk/cosmos/azure-cosmos/src/samples/java/com/azure/cosmos/ChangeFeedProcessorCodeSnippet.java [hunk 2]
```diff
@@ -2,25 +2,51 @@
...
+                for (JsonNode item : docs) {
+                    // Implementation for handling and processing of each JsonNode item goes here
+                }
             })
             .buildChangeFeedProcessor();
         // END: com.azure.cosmos.changeFeedProcessor.builder
     }
+
+    public void handleChangesCodeSnippet() {
+        String hostName = "test-host-name";
```
Comment (milismsft) [inline sdk/cosmos/azure-cosmos/src/samples/java/com/azure/cosmos/ChangeFeedProcessorCodeSnippet.java:33]: see previous comment

File: sdk/cosmos/azure-cosmos/src/samples/java/com/azure/cosmos/ChangeFeedProcessorCodeSnippet.java (hunk 1 above)
Comment (kushagraThapar) [reply sdk/cosmos/azure-cosmos/src/samples/java/com/azure/cosmos/ChangeFeedProcessorCodeSnippet.java:13]: This doesn't get referenced in the docs at all.. This was just to fix the javadocs warnings.

File: sdk/cosmos/azure-cosmos/src/samples/java/com/azure/cosmos/ChangeFeedProcessorCodeSnippet.java (hunk 2 above)
Comment (kushagraThapar) [reply sdk/cosmos/azure-cosmos/src/samples/java/com/azure/cosmos/ChangeFeedProcessorCodeSnippet.java:33]: This doesn't get referenced in the docs at all.. This was just to fix the javadocs warnings.

Comment (kushagraThapar) [reply sdk/cosmos/azure-cosmos/src/samples/java/com/azure/cosmos/ChangeFeedProcessorCodeSnippet.java:33]: Docs only show what is between BEGIN and END tags

File: sdk/cosmos/azure-cosmos/src/samples/java/com/azure/cosmos/ChangeFeedProcessorCodeSnippet.java (hunk 1 above)
Comment (kushagraThapar) [reply sdk/cosmos/azure-cosmos/src/samples/java/com/azure/cosmos/ChangeFeedProcessorCodeSnippet.java:13]: Docs only show what is between BEGIN and END tags

**Comment by kushagraThapar** [comment]
Screen Shot 2020-06-09 at 6 13 56 PM
Screen Shot 2020-06-09 at 6 36 14 PM

**Pull request merged**